src/libraries/System.Speech/src/Recognition/RecognizerBase.cs

```diff
@@ -1687,8 +1687,6 @@ private void SetSapiGrammarProperties(SapiGrammar sapiGrammar, Uri uri, string r
             ActivateRule(sapiGrammar, uri, ruleName);
         }
 
-        // Method called on background thread to do actual grammar loading.
```

Comment:
The above comment should not be removed

Reply:
The comments looked like a justification for the suppression, addressed cbec252222b5eb18375915f51d89208fb8606ca8